src/network/networkd-ndisc.c

```diff
@@ -444,6 +457,16 @@ static int ndisc_router_process_onlink_prefix(Link *link, sd_ndisc_router *rt) {
         if (r < 0)
                 return log_link_error_errno(link, r, "Failed to get prefix lifetime: %m");
 
+        r = sd_ndisc_router_get_lifetime(rt, &router_lifetime);
+        if (r < 0)
```

Comment:
same as above . We should not return if this value is not here and do the same as it was doing before.